### DISCUSSION TITLE
Reakit → Ariakit Migration Guide?

### DISCUSSION THREAD

**Discussion by dandean** [ariakit/ariakit#3809] [Questions]
Hello – thank you for all of your work and support of this tool.

I'm looking for a migration guide from Reakit to Ariakit. There is a previous discussion, https://github.com/ariakit/ariakit/discussions/1565, which is marked as "answered", but it links to a [markdown file](https://gist.github.com/diegohaz/bc07491aee61a5f2469574b38c5c1aa0#how-to-migrate-from-reakit) which says "there will be a migration guide when we get closer to an official release", which in itself does not answer the question.

Any guidance here would be appreciated.

**Comment by diegohaz** [comment]
Hi! We still don't have a migration guide from Reakit.

If you can leave code snippets that represent more or less how you're using Reakit, I'm glad to refactor them into Ariakit.
